ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/Address.java

```diff
@@ -2,16 +2,17 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Data;
+import lombok.experimental.SuperBuilder;
 import lombok.extern.jackson.Jacksonized;
 import uk.gov.hmcts.ccd.sdk.api.ComplexType;
 
 @AllArgsConstructor
-@Builder
+@SuperBuilder
 @Data
 @Jacksonized
 @ComplexType(name = "Address", generate = false)
+@Deprecated
```

Comment:
so we will be getting rid of this ? and only have the new Address type ?

Reply:
It's deprecated in CCD in favour of the sub classes. There is no difference in JSON structure it's just how XUI handles it.